src/test/java/guitests/guihandles/HelpWindowHandle.java

```diff
@@ -27,6 +27,7 @@ public static boolean isWindowPresent() {
 
     /**
      * Returns the {@code URL} of the currently loaded page.
+     * @TODO: 14/3/2018 Remove this when no longer required
```

Comment:
Remember to clear your TODOs if they are done (I don't know if this is applicable or what this TODO is doing)
Remember to make TODOs obvious to anyone seeing your code too -- good habit to have

Reply:
Will update in my commit